SPECS/libcriterion/libcriterion.spec

```diff
@@ -0,0 +1,56 @@
+%define pr_name criterion
+Summary:        A cross-platform C and C++ unit testing framework for the 21st century
+Name:           lib%{pr_name}
+Version:        2.4.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Libraries
+URL:            https://github.com/Snaipe/Criterion
+Source:         https://github.com/Snaipe/Criterion/releases/download/v%{version}/%{pr_name}-%{version}.tar.xz
+BuildRequires:  clang
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  libffi-devel
+BuildRequires:  make
+BuildRequires:  meson
+BuildRequires:  ninja-build
+Requires:       gcc
+
+%description
+A dead-simple, yet extensible, C and C++ unit testing framework.
+Full documentation: http://criterion.readthedocs.org/
+
+%prep
+%setup -q -n %{pr_name}-%{version}
+
+%build
+%meson
+%ninja_build -C %{_target_platform}
+
+%install
+%ninja_install -C %{_target_platform}
+
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%{_libdir}/*.so*
+%{_libdir}/*.a*
+%{_lib64dir}/*.a
```

Comment:
Is there some configuration we can do to make sure files aren't placed in `%{_lib64dir}`? I know it's symlinked to `%{_libdir}`, but historically we've run into issues when we package things like this.